test/lucas_primes.c

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2018 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+#include <openssl/bn.h>
+#include "testutil.h"
+
+
+/*
+ * List of strong Lucas-Selfridge pseudoprimes lower then 1000000
+ * This is sequence https://oeis.org/A217255
+ */
+static int slpsp[] =
+    { 5459, 5777, 10877, 16109, 18971, 22499, 24569, 25199, 40309, 58519, 75077,
+      97439, 100127, 113573, 115639, 130139, 155819, 158399, 161027, 162133,
+      176399, 176471, 189419, 192509, 197801, 224369, 230691, 231703, 243629,
+      253259, 268349, 288919, 313499, 324899, 353219, 366799, 391169, 430127,
+      436409, 455519, 487199, 510479, 572669, 611399, 622169, 635627, 636199,
+      701999, 794611, 835999, 839159, 851927, 871859, 875879, 887879, 895439,
+      950821, 960859 };
+
+#define N 10000
```

Comment:
Is run time the reason for limiting the testing here?
Otherwise I don't see a need for the conditional pre-processing.

Is there a way to increase the limit when we're running the extended tests?

Reply:
Yes, it's about the runtime. I think with the current settings it takes a few seconds.